modules/xcm-interface/src/lib.rs

```diff
@@ -246,4 +248,39 @@ pub mod module {
 			Self::xcm_dest_weight_and_fee(XcmInterfaceOperation::ParachainFee(Box::new(location))).1
 		}
 	}
+
+	impl<T: Config> ProxyXcm<T::AccountId> for Pallet<T> {
+		/// Cross-chain transfer staking currency to sub account on relaychain.
+		fn transfer_proxy(real: T::AccountId, new_owner: T::AccountId) -> DispatchResult {
+			let (xcm_dest_weight, xcm_fee) = Self::xcm_dest_weight_and_fee(XcmInterfaceOperation::ProxyTransferProxy);
```

Comment:
do we need some migration on runtime upgrade, or manual issue tx after runtime upgrade?

Reply:
We could do a manual storage migration, we could also use general council to call `update_xcm_dest_weight_and_fee` when we want feature to be live